REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.6
+julia 0.7-alpha2
```

Comment:
If those are the only changes, I don't think this is necessary.

Reply:
Sure